### ISSUE TITLE
.NET Core version

### ISSUE THREAD

**Issue by dariuszwit** [TonCunha/multi-microsoft-teams#34]
Hello,

I cannot find the .NET Core version I should install.

Your help with this issue would be appreciated.

Regards,
Dariusz

**Comment by dariuszwit** [comment]
I am asking because I cannot run Multi Teams on my Windows 11. I recevie the message:
To run this application, you must install .NET Core.
<img width="256" alt="image" src="https://user-images.githubusercontent.com/13206374/216004068-d57079f7-a1ca-4765-a2c1-8feebeaa2720.png">
When i click "Yes", nothing happens. .NET Core 6.0 and 7.0 are installed.

Google doesn't say too much about this issue.

**Comment by dariuszwit** [comment]
C:\>dotnet --list-sdks
3.1.426 [C:\Program Files\dotnet\sdk]
5.0.408 [C:\Program Files\dotnet\sdk]
6.0.402 [C:\Program Files\dotnet\sdk]
6.0.405 [C:\Program Files\dotnet\sdk]
7.0.102 [C:\Program Files\dotnet\sdk]

It's working with:
https://dotnet.microsoft.com/en-us/download/dotnet/3.1

<img width="294" alt="image" src="https://user-images.githubusercontent.com/13206374/216007692-e959055e-4eef-4cc5-a51f-e0caca5f3488.png">

**Comment by TonCunha** [comment]
Hi @dariuszwit
Yes, dotnet 3.1 is required. When I release a new version, I will update the .net version to 6.

Regards,
Cleriton Cunha

**Comment by dariuszwit** [comment]
Thank you @TonCunha! I appreciate your help.
Thank you for this great tool! 👍

**Comment by DoKnocK** [comment]
@TonCunha Thanks Cleriton.
@dariuszwit Do you have issues with Teams meeting add-in while using this ? 

Default profile - Teams which I actually use for work, it's primary and use to setup meetings etc with different clients.
Profile # 1 - Client teams which I only use to 1:1 chat with them.

Right now am only launching 2 teams profile as mentioned above to test this but having issues with Teams meeting add-in.

Somehow, my Outlook chooses to insert meeting details from my "Profile # 1" and not the Default profile which its supposed to. Any idea ?

**Comment by dariuszwit** [comment]
@DoKnocK I have not noticed the issue like yours. Maybe it happened but I do not use calendar in Outlook very often. I suppose this is your goal: sync Teams meetings with Outlook calendar.

However, I have just noticed that when I pushed the button: Share to Teams in my Outlook, on account connected with the second instance of Teams (the@seconddomain.com), it showed me the message that I should change the Teams account/instance to the@seconddomain.com.
Of course I am logged to both instances: the@deafultdomain.com and the@seconddomain.com.

![image](https://user-images.githubusercontent.com/13206374/216739548-4cf7a9ea-24ff-4aff-af57-2e53bf867da2.png)

When I click the last button, it opens the default instance of Teams witch is connected with the@deafultdomain.com. Not the the@seconddomain.com instance. 
It looks like Outlook can see only the default instance of Teams.

Probably, this issuse belongs to another topic.
